docs/source/extensions/extension_visualization.rst

```diff
@@ -144,30 +144,90 @@ smoothing algorithm.
 .. figure:: https://raw.githubusercontent.com/asreview/asreview-visualization/master/docs/progression.png
    :alt: Progression
 
-API
----
 
-To make use of the more advanced features, you can also use the
-visualization package as a library. The advantage is that you can make
-more reproducible plots where text, etc. is in the place *you* want it.
-Examples can be found in module ``asreviewcontrib.visualization.quick``.
-Those are the scripts that are used for the command line interface.
+Plotting API
+------------
+
+To make use of the more advanced features and/or incorporate plotting 
+into code, you can use the visualization package as a library using the 
+build-in API.
+
+Setting up a plot
+~~~~~~~~~~~~~~~~~
```

Comment:
I prefer wordings like "Basic usage", "Advanced usage", "For beginners". This is more in line with other open source projects.